pip-dep/requirements.txt

```diff
@@ -1,13 +1,13 @@
 Flask>=1.0.2
 flask_socketio>=3.3.2
-syft-proto>=0.1.0a1
+git+git://github.com/sukhadj/syft-proto@master#egg=syft-proto
```

Comment:
Does this depend on changes in `syft-proto`? If not, we should probably leave the requirement version specification.

Reply:
Yes. I have added GradFunc proto code in proto.json
https://github.com/sukhadj/syft-proto/blob/88b27455905264e9d3e11950936ab2b7e5493aed/proto.json#L159

Reply:
Ah, okay, cool. Could you open a `syft-proto` PR for that change?

Reply:
Yes